multi/stm32l1-multi/stm32-multi.h

```diff
@@ -162,7 +162,7 @@ typedef struct {
 enum { spi1 = 0, spi2, spi3 };
 
 
-enum { spi_cmd = 0x1, spi_address = 0x2, spi_dummy = 0x4 };
+enum { spi_cmd = 0x1, spi_dummy = 0x2, spi_addrshift = 2, spi_addrmask = 0x7, spi_addrlsb = 0x20 };
```

Comment:
spi_addrshift and spi_addrmask should not really be a part of the same enum as other flags - these two symbols have very different meaning. I think it is justified to make them defines; SPI_ADDRSHIFT and SPI_ADDRMASK

Reply:
I agree that they have different meaning, but should we at least insert something like a `/* SPI_ADDR */` placeholder comment to explicitly mark these bits reserved or is `spi_addrlsb = 0x20` enough to deter one from using these?

Reply:
Sure, that's great idea to place comment there.

Reply:
Done